packages/itwinui-css/src/checkbox/checkbox.scss

```diff
@@ -72,17 +72,8 @@
   border-radius: var(--iui-border-radius-1);
   background-color: var(--_iui-checkbox-background-color);
   cursor: pointer;
-
-  &::before {
-    content: '';
-    position: absolute;
-    inset: 0;
-    transition: border-color var(--iui-duration-1) ease-out;
-    border-radius: inherit;
-    border-style: solid;
-    border-width: 1px;
-    border-color: var(--_iui-checkbox-border-color);
-  }
+  transition: border-color var(--iui-duration-1) ease-out;
+  border: solid 1px var(--_iui-checkbox-border-color);
```

Comment:
If you check the bottom of the PR you'll see that visual tests are failing.  If you click the circled link it will take you [here](https://itwin.github.io/iTwinUI/1096/results/html_report/) where you can see what visuals changed (under the 64 FAILED tab).
![Screenshot 2023-02-24 at 11 34 51 AM](https://user-images.githubusercontent.com/849817/221234831-afa4bd56-c025-4d47-a10f-3488f8c0e25b.png)

So it would appear that adding a border is making the checkmark symbol smaller.  I think a way to fix this would be to use `outline` instead of `border` like so:
```diff
- transition: border-color var(--iui-duration-1) ease-out;
+ transition: outline-color var(--iui-duration-1) ease-out;
- border: solid 1px var(--_iui-checkbox-border-color);
+ outline: solid 1px var(--_iui-checkbox-border-color);
+ outline-offset: -1px;
```

I tested this locally and test images are passing with this proposed change.

Reply:
I believe we went with a psuedo-element because this is how it looks in Safari:
![](https://user-images.githubusercontent.com/9084735/221694457-c6a3b646-d548-42f6-b2fa-457e7cd7268a.png)

but that will fix automatically in Safari 16.4, so this is fine imo